tests/unit/test_spds_agent.py

```
@@ -45,7 +46,7 @@ def mk_agent_state(
 
 from letta_client.errors import NotFoundError
 
-from spds import config
+from spds import config, tools
```

Flake8 (7.2.0)

[error] 'spds.config' imported but unused

(F401)